### PR TITLE
Add OWNERS files

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - dariadomagala
+    - akucharska
+    - sjanota
+    default-approvers:
+    - parostatkiem
+    - kwiatekus
+    - y-kkamil
+    - dariadomagala
+    - akucharska
+    content-reviewers:
```
Comment (mmitoraj) [inline OWNERS_ALIASES]: For me `content` is quite misleading as it may suggest docu ;) Maybe `peer-reviewers` would be better?

Comment (kazydek) [inline OWNERS_ALIASES]: IMHO, that somehow suggests doc-related reviews.

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    - magicmatatjahu
+    - aerfio
+    - m00g3n
```
Comment (aerfio) [inline OWNERS_ALIASES]: Marcin is on holidays, but I think he'd not like to be included here as he has little to no experience in console

File: OWNERS_ALIASES (hunk 1 above)
Comment (sjanota) [reply OWNERS_ALIASES]: `content` is a directory in this repository. That's why we used that name. Do you still think it's better to name it differently?

**Pull request merged**